qemu/tests/block_copy.py

```diff
@@ -428,3 +428,44 @@ def verify_md5(self, file_name):
         if status != 0:
             raise error.TestFail("File %s changed, md5sum check output: %s" %
                                  (file_name, output))
+
+    def mount_disks(self):
+        """
+        Mount added disks inside guest.
+        """
+        os_type = self.params["os_type"]
+        if os_type == "linux":
```

Comment:
1. There is mount() in virttest/utils_misc.py, but which is used for host, we could improve it for guest too.
just like: https://github.com/avocado-framework/avocado-vt/pull/1077 .
2. For format disk, we could discuss with @suqinhuang for more further improvements.